config/config.go

```diff
@@ -66,6 +66,34 @@ func NewLocal(cfgPath, basepath string) (*Local, error) {
 	return cfg, nil
 }
 
+// Validate validates the values of given configuration
+func (l *Local) Validate() []error {
+	var errors []error
+
+	if err := l.validateIssueTracker(); err != nil {
+		errors = append(errors, err)
+	}
```

Comment:
I like this approach with the `[]errors`. The only thing I would change in the future is invoking the validation funcs like this:
```
func (l *Local) validateFuncs(fs validateFunc) []errors { ... }

Then, the usage would look like:
func (l *Local) Validate() []errors {
    return validateFuncs(validateIssueTracker, validateFoo, validateBar)
}
```

No need for changes now as it is only a single validation func, but wanted to hear your thoughts.

Reply:
Sure! I think it's a great way to deal with them. maybe we could do it if we had 3 or more validation functions to run! :+1: